src/unix/ksignalhandler.hh

```diff
@@ -0,0 +1,68 @@
+/*
+    SPDX-FileCopyrightText: 2021 Aleix Pol Gonzalez <aleixpol@kde.org>
+
+    SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Does this license compatible with GPL3

Reply:
Answer is YES.
https://opensource.stackexchange.com/questions/7977/is-lgpl-compatible-with-gplv3#:~:text=Yes%3B%20both%20LGPLv2.1%20and%20LGPLv3%20are%20compatible%20with,with%20the%20various%20versions%20of%20the%20GNU%20GPL.

Reply:
The "LGPL-2.1-or-later" section 3 says

>  3. You may opt to apply the terms of the ordinary GNU General Public License instead of this License to a given copy of the Library. To do this, you must alter all the notices that refer to this License, so that they refer to the ordinary GNU General Public License, version 2, instead of to this License. (If a newer version than version 2 of the ordinary GNU General Public License has appeared, then you can specify that version instead if you wish.) Do not make any other change in these notices. 

And this footnote https://www.gnu.org/licenses/gpl-faq.en.html#compat-matrix-footnote-7 in GNU's FAQ

Based on the text, we should change this file's license to GPLv3, as allowed by "LGPL-2.1-or-later".